.devcontainer/Dockerfile

```diff
@@ -49,4 +49,6 @@ USER $USERNAME
 RUN wget https://github.com/robbyrussell/oh-my-zsh/raw/master/tools/install.sh -O - | zsh || true
 
 # Install rust
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
\ No newline at end of file
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+RUN echo "[ -f /opt/intel/sgxsdk/environment ] && . /opt/intel/sgxsdk/environment" >> ~/.zshenv
```

Comment:
This was added so that all the tests would run in the container, without it there would be an error linking in the urts.so for the urts tests.